1.1/terms_and_defns.ttl

```diff
@@ -0,0 +1,159 @@
+@prefix cs: <http://www.opengis.net/def/geosparql/td> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix td: <http://www.opengis.net/def/geosparql/td/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+cs:
+    a owl:Ontology , skos:ConceptScheme ;
```

Comment:
Why is this an `owl:Ontology` too?

Reply:
Some tooling (e.g. TopBraid stuff) likes it, but it doesn't have to be. I can strip this out.

Reply:
removed in Commit https://github.com/opengeospatial/ogc-geosparql/pull/165/commits/ecd8b78b1877170f1a34e3fa407aa980cd0978ba